src/extractor.rs

```diff
@@ -113,9 +116,11 @@ enum ExprT {
     DictT(HashMap<String, ExprT>),
     // built-in function types
     // args represented as positional regardless of kwargs in source
-    RefT(String, Option<String>),
+    RefT(String, Option<String>, Option<RefVersion>),
```

Comment:
If you choose to make a new Ref struct, this would look like:
```suggestion
    RefT(Ref),
```